portfolio/src/main/webapp/script.js

```diff
@@ -16,19 +16,32 @@
  * Toggles between showing and hiding a surprise image.
  */
 function showImage() {
-    var x = document.getElementById("surprisePic");
-    if (x.style.display === "block") {
-        x.style.display = "none";
-    } else {
-        x.style.display = "block";
-    }
+  var x = document.getElementById("surprisePic");
+  if (x.style.display === "block") {
+    x.style.display = "none";
+  } else {
+    x.style.display = "block";
+  }
 }
 
 /**
- * Shows the following greeting: "Hello Hana!"
+ * Shows the list of comments
  */
-async function getGreeting() {
-  const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById('greeting-container').innerText = greeting;
+function getComments() {
+  fetch("/data")
+    .then(response => response.json())
+    .then(comments => {
+      const commentsListElement = document.getElementById('comments-list');
+      commentsListElement.innerHTML = "";
+      commentsListElement.appendChild(createListElement(comments[0]));
+      commentsListElement.appendChild(createListElement(comments[1]));
+      commentsListElement.appendChild(createListElement(comments[2]));
```

Comment:
(Optional): Could also write this like this:
```suggestion
      for (const comment of list) {
        const liElement := document.createElement('li');
        liElement.innerText = comment;
        commentsListElement.appendChild(comment);
      }
```

Reply:
Just noticed this was addressed in PR 4 so nvm :)